.github/scripts/pkg-test.sh

```diff
@@ -14,7 +14,8 @@ install_debian_like() {
 
   # Install Netdata
   # Strange quoting is required here so that glob matching works.
-  apt-get install -y $(find /netdata/artifacts -type f -name 'netdata*.deb' ! -name '*dbgsym*' ! -name '*cups*' ! -name '*freeipmi*') || exit 3
+  apt-get install -y "$(find /netdata/artifacts -type f -name 'netdata*.deb' \
+! -name '*dbgsym*' ! -name '*cups*' ! -name '*freeipmi*')" || exit 3
```

Comment:
```suggestion
  apt-get install -y $(find /netdata/artifacts -type f -name 'netdata*.deb' \
! -name '*dbgsym*' ! -name '*cups*' ! -name '*freeipmi*') || exit 3
```

Reply:
>  Seems error's only for DEB - unrelated

It totally related, you made a change to `install_debian_like` in `pkg-test.sh`. Remove `"`.

Reply:
Disabled this shellcheck